M2/Macaulay2/packages/ResLengthThree.m2

```diff
@@ -0,0 +1,752 @@
+newPackage ( "ResLengthThree",
```

Comment:
This is a new package, so its name should be added to the list in the file =distributed-packages